oak-run/src/main/java/org/apache/jackrabbit/oak/run/AvailableModes.java

```diff
@@ -75,5 +75,6 @@ public final class AvailableModes {
             .put("segment-copy", new SegmentCopyCommand())
             .put("server", new ServerCommand())
             .put("purge-index-versions", new LucenePurgeOldIndexVersionCommand())
+            .put("generate-test-garbage", new GenerateGarbageCommand())
```

Comment:
I would suggest changing `generate` to `create`.
Generate is used with non-tangible things whereas `create` is used with tangible ones.

`create` seems more accurate grammatically.

Reply:
Fixed, thank you for the suggestion.